go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/88250/lute v1.7.3
+	github.com/Masterminds/sprig v2.22.0+incompatible
```

Comment:
LGTM. But the latest stable of `sprig` is v3. could you update for this?

Reply:
Sure!

Reply:
No its done.